src/pudl/extract/ferc1.py

```diff
@@ -903,3 +912,95 @@ def extract_xbrl_metadata(
                 xbrl_meta_out[pudl_table][period] = []
 
     return xbrl_meta_out
+
+
+# DAGSTER ASSETS
+def create_raw_ferc1_assets() -> list[SourceAsset]:
+    """Create SourceAssets for raw ferc1 tables.
+
+    SourceAssets allow you to access assets that are generated elsewhere.
+    In our case, the xbrl and dbf database are created in a separate dagster repository.
```

Comment:
Correct @jdangerx. I agree with @zschira. To convert the ferc_to_sqlite steps to assets we need to know table names ahead of time. I think the DBF tables are pretty static but the XBRL tables can vary. I guess if a new archive of XBRL data introduces a new table we can just update the pipeline accordingly.

We'll also have to tweak the `ferc-xbrl-extractor` to return data frames so the IO Managers can write to the DB instead of writing to the db from within the extract method.